### PR TITLE
Fixing metagames

### PR DESCRIPTION
**Pull request by TheMezStrikes**
__Metas fixed so far__
[Gen 7] Shared Power (Random)
[Gen 7] Dragon Heaven Super Staff Bros
[Gen 7] [Main] Super Staff Bros. Melee

For whatever reason, some of the commits that have already been merged are still included in this PR... I gotta figure out how git works soon

### CODE REVIEW

File: config/formats.js [hunk 1]
```diff
@@ -104,7 +104,7 @@ exports.Formats = [
 	},
 	{
 		name: "[Gen 7] Dragon Heaven Super Staff Bros",
-		desc: ["&bullet; The staff here becomes a Pokemon and battles! <br> &bullet; <a href=\"https://github.com/XpRienzo/DragonHeaven/blob/master/mods/dhssb/README.md\">Movesets</a>"],
+		desc: ["&bullet; The staff here becomes a Pokemon and battles! <br> &bullet; <a href=\"https://github.com/XpRienzo/DragonHeaven/blob/master/mods/dhssb/README.md\">MoveSlotss</a>"],
```
Comment (Spandamn) [inline config/formats.js]: Nice find and replace haha

File: config/formats.js [hunk 2]
```diff
@@ -118,11 +118,12 @@ exports.Formats = [
 			let allPokemon = this.p1.pokemon.concat(this.p2.pokemon);
 			for (let i = 0, len = allPokemon.length; i < len; i++) {
 				let pokemon = allPokemon[i];
+				console.log(pokemon.moveSlots);
```
Comment (Spandamn) [inline config/formats.js]: ??

File: mods/dhssb/random-teams.js [hunk 3]
```diff
@@ -0,0 +1,249 @@
+'use strict';
```
Comment (Spandamn) [inline mods/dhssb/random-teams.js]: Wow cool I wanted to do this a long time ago

File: mods/pokebilities/random-teams.js [hunk 4]
```diff
@@ -0,0 +1,178 @@
...
+			if (set.moves.includes('sunnyday') || set.ability === 'Drought' && !item.onPrimal) teamDetails['sun'] = 1;
+			if (set.moves.includes('stealthrock')) teamDetails['stealthRock'] = 1;
+			if (set.moves.includes('toxicspikes')) teamDetails['toxicSpikes'] = 1;
+			if (set.moves.includes('defog') || set.moves.includes('rapidspin')) teamDetails['hazardClear'] = 1;
+		}
+		return pokemon;
+	}
+}
+
+module.exports = RandomOMTeams;
```
Comment (Spandamn) [inline mods/pokebilities/random-teams.js]: Last line should be blank \n

File: mods/ssbm/pokedex.js [hunk 5]
```diff
@@ -0,0 +1,47 @@
...
+		otherForms: ["missingno"],
+	},
+	
+	// UOP is bot
+	klang: {
+		inherit: true,
+		baseStats: {hp: 90, atk:80, def: 110, spa:80, spd:110, spe: 75},
+		gen: "SSB",
+	},
+	unleashourpassion: {
```
Comment (Spandamn) [inline mods/ssbm/pokedex.js]: Add this to DHSSB imo

File: config/formats.js (hunk 1 above)
Comment (TheMezStrikes) [reply config/formats.js]: oof

File: config/formats.js (hunk 2 above)
Comment (TheMezStrikes) [reply config/formats.js]: Sorry for leaving that in

File: mods/dhssb/random-teams.js (hunk 3 above)
Comment (TheMezStrikes) [reply mods/dhssb/random-teams.js]: Well I did it for you

**Comment by TheMezStrikes** [comment]
Fixed all the problems highlighted in the review

**Pull request merged**